internal/scorecard/alpha/tests/crhelper.go

```diff
@@ -0,0 +1,66 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tests
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"github.com/operator-framework/operator-registry/pkg/registry"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+// GetCRs parses a Bundle's CSV for CRs
```

Comment:
We should ask the OLM team if this is a reasonable method to add to the `registry.Bundle` or CSV structs. If not, this make sense as is.

Reply:
I entered this as a need/example in https://github.com/operator-framework/api/issues/28